### PR TITLE
Update javascript-basics-part-1.md

### PR DESCRIPTION
**Pull request by f0ru0l0rd**
Help clarify points with examples.

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/intro-js/javascript-basics-part-1.md?ref=486c6b01438cf859b799c04c16b97713c3b79767

File: javascript/core/intro-js/javascript-basics-part-1.md [hunk 1]
```diff
@@ -33,9 +33,17 @@ Like most programming languages, JavaScript has a number of rules its code must
 
 Everything in JavaScript is case sensitive.
 
+```javascript
+var password;
+// Is not the same as
+var passWord;
+/or
```
Comment (kapnobatai137) [inline javascript/core/intro-js/javascript-basics-part-1.md]: this should have a double forward slash "//or"

File: javascript/core/intro-js/javascript-basics-part-1.md [hunk 2]
```diff
@@ -33,9 +33,17 @@ Like most programming languages, JavaScript has a number of rules its code must
...
+var PASSWORD;
+```
+
 ### Keywords
 
-Like most languages, there are a number of reserved words in JavaScript that you cannot use to name your functions and variables.
+Like most languages, there are a number of reserved words in JavaScript that you cannot use to name your functions and variables (eg: var, let, new)
```
Comment (kapnobatai137) [inline javascript/core/intro-js/javascript-basics-part-1.md]: There should be a dot at the end of the phrase.

File: javascript/core/intro-js/javascript-basics-part-1.md (hunk 1 above)
Comment (f0ru0l0rd) [reply javascript/core/intro-js/javascript-basics-part-1.md]: Yes, and apologies, I was doing markdown on my phone. Good catch.

Comment (kapnobatai137) [reply javascript/core/intro-js/javascript-basics-part-1.md]: It's not a problem, I just wanted to point these out before we could merge. Thank you for taking the time to improve the content.

**Pull request closed without merging**